fMRISurface/GenericfMRISurfaceProcessingPipeline.sh

```diff
@@ -34,9 +34,9 @@ source "${HCPPIPEDIR}/global/scripts/processingmodecheck.shlib"  # Check process
 
 opts_SetScriptDescription "Run fMRISurface processing"
 
-opts_AddMandatory '--studyfolder' 'Path' 'path' "folder containing all subject" "--path"
+opts_AddMandatory '--studyfolder' 'Path' 'path' "folder containing all session" "--path"
 
-opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
+opts_AddMandatory '--session' 'Session' 'session ID' "--subject"
```

Comment:
This is missing a long description, so the intention of `--subject` to be a legacy flag will not currently work (wrong number of arguments).  Use a quoted empty string, if nothing else.
```suggestion
opts_AddMandatory '--session' 'Session' 'session ID' "" '--subject'
```